e2e/cypress/integration/integrations/builtin_commands/common_commands_spec.js

```diff
@@ -317,6 +317,122 @@ describe('I18456 Built-in slash commands: common', () => {
+        cy.get('#multiSelectList').findByText(`@${userDM.username}`).click();
+        cy.findByText('Go').click();
+        cy.get('#channelHeaderDropdownButton').contains(`${userDM.username}`).should('be.visible');
+
+        // # In a GM Use the /invite command to invite a channel to another channel (e.g., /invite @[channel name])
+        cy.postMessage(`/invite @${testChannel.name}`);
+
+        // * Error appears: "We couldn't find the user. They may have been deactivated by the System Administrator."
+        cy.uiWaitUntilMessagePostedIncludes('We couldn\'t find the user. They may have been deactivated by the System Administrator.');
+    });
+
+    it('MM-T660_2 /invite tests when used in DMs and GMs', () => {
+        const userDM = userGroup[2];
+        const userToInvite = userGroup[3];
+
+        cy.apiAddUserToChannel(testChannel.id, userToInvite.id);
+        cy.apiAddUserToChannel(testChannel.id, user1.id);
+        loginAndVisitDefaultChannel(user1, testChannelUrl);
+        cy.get('#postListContent', {timeout: TIMEOUTS.HALF_MIN}).should('be.visible');
+
+        // # In a GM use the /invite command to invite someone to a channel they're already a member of
+        cy.postMessage(`/groupmsg @${userGroup[0].username} @${userGroup[1].username}`);
+        cy.postMessage(`/invite @${userToInvite.username} ~${testChannel.name}`);
+
+        // * Error appears: "[username] is already in the channel"
+        cy.uiWaitUntilMessagePostedIncludes(`${userToInvite.username} is already in the channel.`);
+
+        cy.get('#addDirectChannel').click();
+        cy.get('#selectItems').type(`${userDM.username}`);
+        cy.wait(TIMEOUTS.ONE_SEC);
+        cy.get('#multiSelectList').findByText(`@${userDM.username}`).click();
+        cy.findByText('Go').click();
+        cy.get('#channelHeaderDropdownButton').contains(`${userDM.username}`).should('be.visible');
+
+        // # In a DM use the /invite command to invite someone to a channel they're already a member of
+        cy.postMessage(`/invite @${userToInvite.username} ~${testChannel.name}`);
+
+        // * Error appears: "[username] is already in the channel"
+        cy.uiWaitUntilMessagePostedIncludes(`${userToInvite.username} is already in the channel.`);
+    });
+
+    it('MM-T660_3 /invite tests when used in DMs and GMs', () => {
+        const userA = userGroup[0];
+        const userB = userGroup[1];
+        const userC = userGroup[2];
+        const userDM = userGroup[3];
+
+        // # As UserA create a new public channel
+        loginAndVisitDefaultChannel(userA, `${team1.name}/channels/town-square`);
+        cy.get('#postListContent', {timeout: TIMEOUTS.TWO_MIN}).should('be.visible');
+        cy.get('#createPublicChannel').click();
+        cy.get('#newChannelName').type(`${userA.username}-channel`);
+        cy.get('#submitNewChannel').click();
+        cy.get('#postListContent').should('be.visible');
+
+        cy.apiLogout();
+        cy.apiLogin(userB);
```

Comment:
Failed on me. I think it needs to visit a channel or reload the page after logged in.
```suggestion
        cy.apiLogin(userB);
        cy.reload();
```